libs/ledger-live-common/src/bot/portfolio/formatter.ts

```diff
@@ -282,6 +283,48 @@ function formatSize(bytes: number | undefined): string | undefined {
   return kb < 1024 ? `${kb.toFixed(0)}kb` : `${(kb / 1024).toFixed(0)}mb`;
 }
 
+export function urlToEndpoint(url: string): string {
+  // eslint-disable-next-line no-useless-escape
+  const regex = new RegExp(/^(?:https?:\/\/)?(?:[^@\/\n]+@)?(?:www\.)?([^:\/\n]+)/, "igm");
+  const regexResult = regex.exec(url);
+  return regexResult ? url.slice().replace(regexResult[0], "") : url;
+}
+
+export function formatDetails(
+  details: { [endpoint: string]: NetworkAuditDetails } | undefined,
+): string | undefined {
+  if (!details) return;
+  let report = "";
+  const boldIfHigherThan1 = nbr => (nbr > 1 ? `**${nbr}**` : nbr);
```

Comment:
As discussed: this could be changed to filter-out results that aren't `> 1`

Name of the section should then be restored to "Duplicate HTTP Calls"

Reply:
Done